app/app_test.go

```diff
@@ -55,8 +55,12 @@ func TestFyneApp_OpenURL(t *testing.T) {
 
 func TestFyneApp_SendNotification(t *testing.T) {
 	n := fyne.NewNotification("Test Title", "Some content")
+	testApp := test.NewApp()
+	oldApp := fyne.CurrentApp() // we are testing the app package so don't use the test app
+	fyne.SetCurrentApp(testApp)
+	defer fyne.SetCurrentApp(oldApp)
```

Comment:
Looks strange. You
* build a new test app which is implicitly set as current app -> `testApp`
* get the current app which is at this point the same as `testApp` -> `oldApp == testApp`
* set the `testApp` as current app which does nothing because it already is
* defer setting `oldApp` as current app which doesn’t change anything because `oldApp == testApp`

Reply:
Or more simply "I think line 58 amd 59 are the wrong way round".
Agreed, apologies.